web/controllers/rules/pr/reviewer_but_no_assignee.ex

```diff
@@ -0,0 +1,78 @@
+defmodule Dwylbot.Rules.PR.ReviewerButNoAssignee do
+  @moduledoc """
+  Check for error when an PR has awaiting-review but no assignees
+  """
+  alias Dwylbot.Rules.Helpers
+  @github_api Application.get_env(:dwylbot, :github_api)
+  @rule_name "pr_reviewer_but_no_assignee"
+
+  def apply?(payload) do
+    reviewers = payload["pull_request"]["requested_reviewers"]  |> Enum.map(&(&1["login"]))
+    payload["action"] in ~w(labeled unassigned assigned) && !Enum.empty?(reviewers)
+  end
+
+  def check(payload, _get_data?, token) do
```

Comment:
I think we are missing some logic. We need to get the data of the PR after 30s and check for the error again otherwise dwylbot will comment even if a assignee has been added to the PR